app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -1,17 +1,34 @@
-# Thank you for completing your teacher training application
+Dear <%= @application_form.first_name %>,
 
-You have submitted an application to:
+# Application submitted
+
+You have successfully submitted an application for:
+
+<% @application_form.application_choices.each do |application_choice| %>
+* <%= application_choice.provider.name %> - <%= application_choice.course.name %>
+<% end %>
+
+Your application reference is <%= @application_form.support_reference %>.
+
+You can track the progress of your <%= 'application'.pluralize(@application_form.application_choices.count) %> here:
```

Comment:
it would be nice to have all these `pluralize` into helpers or maybe a Email Component,
anyway not a priority now.